### DISCUSSION TITLE
No Rubberduck status bar

### DISCUSSION THREAD

**Discussion by doobes** [rubberduck-vba/Rubberduck#6224] [Q&A] [answered]
I'm using Excel 2010.

Downloaded the latest Rubberduck.  Installed as an Admin and rebooted.

Open Excel, save file as a macro (.xlsm)

Open the IDE, no RubberDuck:

![image](https://github.com/user-attachments/assets/fcfff12d-71b9-4382-b329-7bcebc8225c6)

Help!

**Comment by INOPIAE** [comment]
Use Add-Ins menue in the IDE to see if Rubberduck is installed, loaded and started.

**Reply by doobes** [reply, marked as answer]
Not sure what happened, but it's there now. Thanks.
